pymemcache/test/test_client.py

```diff
@@ -52,6 +54,74 @@ def is_ipv6(address):
     return re.match(r'^[0-9a-f:]+$', address)
 
 
+@pytest.mark.unit()
+def test_check_key_helper():
+    testing_matrix = (
```

Comment:
This would be a good place to use [`pytest.mark.parametrize`](https://docs.pytest.org/en/6.2.x/parametrize.html#parametrize).